files/helpers.py

```diff
@@ -669,8 +686,8 @@ def produce_ffmpeg_commands(media_file, media_info, resolution, codec, output_fi
     else:
         return False
 
-    src_framerate = media_info.get("video_frame_rate", 30)
-    if src_framerate <= 30:
+    target_fps = Fraction(int(media_info.get("video_frame_rate_n", 30)), int(media_info.get("video_frame_rate_d", 1)))
```

Comment:
Instead of defaulting to 30 here, I would rather let ffmpeg encode the input as-is. Fixing it to 30 could introduce unwanted frame duplications.

Reply:
Yes, that is one option. The other is to think about the situation when this number is used. If I am not mistaken that happens only if the "video_frame_rate_n" value fails to be extracted by FFprobe. Then I guess the source is somehow broken or in a weird format, simply not standard. The question is what is better: trying to change the framerate to some standard value or letting FFmpeg decide. So basically I think it is frame duplication vs compatibility/non-deterministic result. I do not have a preference, either has its own benefits and downsides. Or am I wrong?